auditbeat/buildkite.yml

```diff
@@ -0,0 +1,50 @@
+when:
+    changeset:                 ## when PR contains any of those entries in the changeset
+        - "^auditbeat/.*"
+        - "@ci"                ## special token regarding the changeset for the ci
+        - "@oss"               ## special token regarding the changeset for the oss
+stages:
+    # mandatory stage - it runs always for:
+    #   - branches/tags
+    #   - on PRs
+    #   - GitHub comment /test auditbeat
+    #   - GitHub label auditbeat
+    mandatory:
```

Comment:
All the stages in the `mandatory` meta-stage will be triggered for PRs using:
* if changes in the changeset.
* The GitHub comment `/test auditbeat`
* The GitHub label `mandatory-auditbeat`

Reply:
How is the triggers configured from this yaml, I thought that this is something that we need to configure as part of the buildkite-pr-bot.

Reply:
>  I thought that this is something that we need to configure as part of the buildkite-pr-bot.

That's correct, but  `buildkite-pr-bot.` will need to be configured with a meta-regex:
- https://github.com/elastic/beats/pull/38173 (for the time being I'm using `buildkite test` rather than `/test` so, we can test this in isolation.

Then the pipeline generator will be able to enable/disable any specific stages dynamically afterwards.

Reply:
> How is the triggers configured from this yaml

See https://github.com/elastic/beats/pull/38162/commits/6b368115d2556dd74b85da652789fbe284e0a505

It uses the `GITHUB_PR_TRIGGER_COMMENT` and `GITHUB_PR_LABELS` env variables provided by the `buildkite-pr-bot`

Reply:
oh, ok it make sense, thanks